make_lightgbm.sh

```diff
@@ -20,7 +20,8 @@ set -e
 
 # Usage: $1 serves to checkout a specific commit/tag
 
-git clone --recursive https://github.com/microsoft/LightGBM ; cd LightGBM
+echo "Cloning $LIGHTGBM_REPO_URL..."
+git clone --recursive "$LIGHTGBM_REPO_URL" ; cd LightGBM
```

Comment:
Is this idempotent? Meaning that if the repo is already cloned (because I've ran this in the past) will it still work?

Reply:
Yes, because it happens inside the fresh docker environment.